### ISSUE TITLE
Allow functions with no parameters

### ISSUE THREAD

**Issue by benjamincharity** [AndyOGo/stylelint-declaration-strict-value#39]
Most of our Sass functions have default parameters, but using functions with no parameters is reported as an error.

#### Example

With `ignoreFunctions: true`

```scss
// No error
.foo {
  padding: spacing(large);
}

.bar {
  padding: spacing();
}
```

Currently I'm moving forward by defining the default parameter in all function calls, but I know our core teams aren't going to like a requirement to type more just for the linter.

**Comment by AndyOGo** [comment]
Hy @benjamincharity 
Thank you.
Nice catch, I will look into as soon as I have some free time.

**Comment by AndyOGo** [comment]
@benjamincharity Sorry for the late fix, I was very busy codingfor other projects and totally forgot about it.
I just added tests and it should work as expected now. Thanks again for your valueable feedback.
